templates/project.html

```diff
@@ -253,10 +268,12 @@ <h3 class="question-group-title">{{group.title}}</h3>
                     {# The layout of the project page is either a grid of components or a vertical list of components. #}
                     {% if layout_mode == "grid" %}col-sm-3 col-lg-2{% else %}col-xs-12{% endif %}
                     question
+                    question-row-container
```

Comment:
No need for this extra class, per above.

Reply:
My bad. Forgot to remove that css.